langserver/langserver.py

```diff
@@ -101,26 +129,22 @@ def serve_initialize(self, request):
             }
         }
 
-    def serve_hover(self, request):
+    def serve_hover(self, request, parent_span):
         params = request["params"]
         pos = params["position"]
         path = path_from_uri(params["textDocument"]["uri"])
-        source = self.fs.open(path)
+        source = self.fs.open(path, parent_span)
```

Comment:
Shouldn't block this PR getting out, but in the future as part of getting some version of our FS into jedi, this call signature won't be accepted. I would expect what we do instead is something which wraps an `fs` with opentracing logging.